packages/git/src/browser/git-contribution.ts

```diff
@@ -743,6 +449,83 @@ export class GitContribution implements
             command: GIT_COMMANDS.OPEN_CHANGES.id,
             tooltip: GIT_COMMANDS.OPEN_CHANGES.label
         });
+
+        const registerItem = (item: Mutable<TabBarToolbarItem>) => {
+            const commandId = item.command;
+            const id = '__git.tabbar.toolbar.' + commandId;
+            const command = this.commands.getCommand(commandId);
+            this.commands.registerCommand({ id, iconClass: command && command.iconClass }, {
+                execute: (widget, ...args) => widget instanceof ScmWidget && this.commands.executeCommand(commandId, ...args),
+                isEnabled: (widget, ...args) => widget instanceof ScmWidget && this.commands.isEnabled(commandId, ...args),
+                isVisible: (widget, ...args) => widget instanceof ScmWidget && this.commands.isVisible(commandId, ...args),
+            });
+            item.command = id;
+            registry.registerItem(item);
+        };
+
+        registerItem({
+            id: GIT_COMMANDS.COMMIT.id,
+            command: GIT_COMMANDS.COMMIT.id,
+            tooltip: GIT_COMMANDS.COMMIT.label
+        });
+        registerItem({
+            id: GIT_COMMANDS.REFRESH.id,
+            command: GIT_COMMANDS.REFRESH.id,
+            tooltip: GIT_COMMANDS.REFRESH.label
+        });
+        registerItem({
+            id: GIT_COMMANDS.COMMIT_ADD_SIGN_OFF.id,
+            command: GIT_COMMANDS.COMMIT_ADD_SIGN_OFF.id,
+            tooltip: GIT_COMMANDS.COMMIT_ADD_SIGN_OFF.label
```

Comment:
... which is a toggle, right? it also removes it.

Reply:
since it just was moved to toolbar, i think we can leave it as is